src/automate.js

```diff
@@ -1,7 +1,7 @@
 const {MemoryModel, config} = require("./memory_model.js");
 
 /**
- * Draws the objects given in the path in an automated fashion.
+ * Draws the objects given in the path in an automated.md fashion.
```

Comment:
I'm not sure what happened, but you seem to have renamed `automated` to `automated.md` in at least two places. Make sure to revert all such changes